content/en/docs/tasks/job/pod-failure-policy.md

```diff
@@ -242,6 +242,65 @@ kubectl delete jobs/job-pod-failure-policy-config-issue
 
 The cluster automatically cleans up the Pods.
 
+## Using Pod Failure Policy to avoid unnecessary Pod retries per index
+
+To avoid unnecessary Pod restarts per index, you can use the Pod failure policy and
+Backoff limit per index features. The following section shows how to use these features:
+
+1. Save the following manifest as `job-backoff-limit-per-index-failindex.yaml`:
```

Comment:
We prefer to recommend fetching manifests via HTTPS
```suggestion
1. Examine the following manifest:
```

It does feel a bit `curl | bash` but we also promise that the manifest looks like the one in the page (and I think that's a reasonable promise.

Reply:
yeah, I'm happy to update just thinking what to do to keep the doc consistent throughout, as in the other comment: https://github.com/kubernetes/website/pull/49811#discussion_r2014519065

Reply:
Addressed tentatively in the latest commit: https://github.com/kubernetes/website/pull/49811/commits/d1eb544f18c29ae024d7ef0bf4426973061d5c54
It makes the diff bigger, but I like making the sections consistent. PTAL.